sftp.c

```diff
@@ -1531,7 +1538,7 @@ parse_args(const char **cpp, int *ignore_errors, int *disable_echo, int *aflag,
 		if (argc - optidx < 1)
 			goto need_num_arg;
 		errno = 0;
-		ll = strtoll(argv[optidx], &cp2, base);
+		ll = strtoll(argv[optidx], &cp2, base); // CodeQL [SM02313]: false positive cp2 will not be uninitialized
```

Comment:
Let's add a check for the `ll` returned variable, and if `0` then `cp2` may be null or uninitialized.